src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2165,6 +2165,17 @@ bool AdaptDispatch::EnableAlternateScroll(const bool enabled)
     return success;
 }
 
+//Routine Description:
```

Comment:
nit: comment format should have spaces after the `/`

Reply:
You know I copied it from the other methods in this file. Take a look at them then you know why it looks like this.

Reply:
OH. Okay. I don't care then! I won't make you fix that :smile: